src/index.html

```diff
@@ -1,13 +1,431 @@
+                    Max Henry
+                  </a>
+                </h3>
+
+                <div class="testimonial__role">
+                  Chief Design Officer
+                </div>
+
+                <p class="testimonial__about">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 234 5555-55-55"
+                  class="contact-us__info-link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+                <a
+                  href="mailto:hello@miami.com"
+                  class="contact-us__info-link"
+                >
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Adress
+                </div>
+
+                <a
+                  rel="noopener noreferrer"
+                  href="https://goo.gl/maps/uYXDMeMFpZuq38vo6"
+                  class="contact-us__info-link"
+                  target="_blank"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--8-12">
+              <form action="get">
+                <input
+                  type="text"
+                  name="name"
+                  placeholder="Name"
+                  class="contact-us__field form-field"
+                  required
+                >
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="email@example.com"
+                  class="contact-us__field form-field"
+                  required
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="contact-us__field contact-us__field--last form-field form-field--textarea"
+                >
+                </textarea>
```

Comment:
I cant see placeholder for this textarea on page https://prnt.sc/1tt4qgj, you should change code style here a bit